wikipediapreview-wordpress.php

```diff
@@ -1,6 +1,10 @@
 <?php
 /**
  * Plugin Name: Wikipedia Preview
- * Description: Add Wikipedia Preview to your articles
- * Version: 0.1.0
+ * Plugin URI: https://github.com/wikimedia/wikipedia-preview
+ * Description: Wikipedia Preview allows you to show a popup card with a short summary from Wikipedia when a reader clicks or hovers over a link
+ * Version: 1.1.1
+ * Author: Wikimedia Foundation
+ * License: MIT
+ * License URI: https://github.com/wikimedia/wikipedia-preview/blob/main/LICENSE
```

Comment:
this should be the plugin info
https://developer.wordpress.org/plugins/plugin-basics/header-requirements/#header-fields

Reply:
Right, those are the available fields. Shall we go through the ones missing?

> Requires at least: The lowest WordPress version that the plugin will work on.
> Requires PHP: The minimum required PHP version.

Should this be the latest versions for WordPress and PHP? 

> Author URI: The author’s website or profile on another website, such as WordPress.org.

We could use https://wikimediafoundation.org/ here
    
> Text Domain: The gettext text domain of the plugin. More information can be found in the Text Domain section of the How to Internationalize your Plugin page.
> Domain Path: The domain path lets WordPress know where to find the translations. More information can be found in the Domain Path section of the How to Internationalize your Plugin page.

I understand we don't need this given the nature of our plugin (no text outside popup) and our use of translatewiki.net 
    
> Network: Whether the plugin can only be activated network-wide. Can only be set to true, and should be left out when not needed.

I don't think this is needed

Reply:
> Right, those are the available fields. Shall we go through the ones missing?
> 
> > Requires at least: The lowest WordPress version that the plugin will work on.
> > Requires PHP: The minimum required PHP version.
> 
> Should this be the latest versions for WordPress and PHP?

Yes

> 
> > Author URI: The author’s website or profile on another website, such as WordPress.org.
> 
> We could use https://wikimediafoundation.org/ here

Right

> 
> > Text Domain: The gettext text domain of the plugin. More information can be found in the Text Domain section of the How to Internationalize your Plugin page.
> > Domain Path: The domain path lets WordPress know where to find the translations. More information can be found in the Domain Path section of the How to Internationalize your Plugin page.
> 
> I understand we don't need this given the nature of our plugin (no text outside popup) and our use of translatewiki.net

We'll need it when we have more UI and translation for it.

> 
> > Network: Whether the plugin can only be activated network-wide. Can only be set to true, and should be left out when not needed.
> 
> I don't think this is needed

I don't understand what it means. We can probably ignore it for now.